Website/pages/about.js

```diff
@@ -22,8 +22,8 @@ export default function about(){
         </div>
         <div className="mt-5 mx-5 border-t-2 border-black">
         <h1 className="text-center font-bold text-3xl py-5">My Stats</h1>
-        <div className="grid grid-cols-2 space-x-2 justify-items-center py-5 pb-20">
-        <Image width={360} height={25}  src="/Resources/github-stats-1.svg" alt=""/>
+        <div className="grid grid-cols-2  lg:-space-x-44 justify-items-center py-5 pb-20">
+        <Image width={360} height={25} className="lg:ml-20"  src="/Resources/github-stats-1.svg" alt=""/>
```

Comment:
Please try to change the stats image with a dynamic image link. You can refer here - https://youtu.be/2U7yZ3wvFBM?t=840 .

Reply:
I have tried adding the dynamic image link but still the image is not showing properly. Its the same as [this comment](https://github.com/SaptarshiSarkar12/Drifty/pull/204#issuecomment-1486126815)

Reply:
ok @naveen13-06 . Leave this one. Work on the rest of the changes asked. I shall try to work on this issue later.